JSONString.java

```diff
@@ -33,21 +33,27 @@ public JSONString(String value) {
    * Convert to a string (e.g., for printing).
    */
   public String toString() {
-    return "";          // STUB
+    return (this.value == null)
```

Comment:
Is `JSONString` nullable? What's the point of JSONConstant then?

Reply:
No lol. For some reason, Rebelsky did that in JSONConstant (which was already implemented) so I mirrored that for consistency